### DISCUSSION TITLE
How to convert FP16 gguf model to 4bit or 5 bit gguf model

### DISCUSSION THREAD

**Discussion by RakshitAralimatti** [ggml-org/llama.cpp#8108] [Q&A] [answered]
I am able to convert the models to fp16 gguf models but facing issue in converting them further in 4bit or 5bit can any one help me to achieve this?

**Comment by micsthepick** [comment]
```
./llama-quantize ../text-generation-webui/models/biblegpt-unsloth/biblegpt-unsloth.F16.gguf  2
```
the above command works for me?
(2 comes from `./llama-quantize --help`)

**Comment by cshamis** [comment, marked as answer]
`% llama-quantize ../models/model_f16.gguf ../models/model_Q4_K_M Q4_K_M`

**Comment by mimi566** [comment]
llama.ccp has it's own llama-quantize features. So check with this ` !ls -lh /content/llama.cpp/bin/`   and then just run `!/content/llama.cpp/bin/llama-quantize /content/your-path-to-f16.gguf  /content/your-path-to-q4_k_m.gguf Q4_K_M`
   Will save here Hope it will save your time.. Thank you..
